src/maidsafe/launcher/ui/models/account_handler_model.cc

```diff
@@ -24,17 +25,33 @@ namespace launcher {
 
 namespace ui {
 
-namespace models {
-
-AccountHandlerModel::AccountHandlerModel(QObject* parent)
-    : QObject{parent} {}
-
-}  // namespace models
+AccountHandlerModel::AccountHandlerModel(QObject* parent) : QObject{parent} {}
+
+AccountHandlerModel::~AccountHandlerModel() = default;
+
+std::unique_ptr<Launcher> AccountHandlerModel::Login(const QString& /*pin*/,
+                                                     const QString& /*keyword*/,
+                                                     const QString& /*password*/) {
+  on_scope_exit sig{[this] { emit LoginResultAvailable(); }};
```

Comment:
If creating the new `Launcher` throws, do we still want to invoke the `LoginResultAvailable()` call?  (Same for `CreateAccount` below?)